doc/howto/network_configure.md

```diff
@@ -12,6 +14,19 @@ lxc network set UPLINK dns.nameservers=8.8.8.8
 
 The available configuration options differ depending on the network type.
 See {ref}`network-types` for links to the configuration options for each network type.
+````
+````{group-tab} UI
+To edit the configuration of a network, navigate to the overview page for the network, and observe its attributes and settings..
+
+Within the Configuration tab, you can edit Bridge, IPV4, IPV5 and DNS settings of the network by clicking on the {guilabel}`Edit` pencil icon inline with the desired configuration setting.
```

Comment:
The available sections differ per network type. Might be better to say you can explore the different sections, than listing the ones for a bridge. Or give a concrete example of DNS nameserver, like the cli does.